src/components/ui/popover/popover.component.tsx

```diff
@@ -241,13 +205,27 @@ export class Popover extends React.Component<PopoverProps, State> {
 
   public render(): React.ReactElement {
     return (
-      <MeasureElement
-        shouldUseTopInsets={ModalService.getShouldUseTopInsets}
-        force={this.state.forceMeasure}
-        onMeasure={this.onChildMeasure}
-      >
-        {this.props.anchor()}
-      </MeasureElement>
+      <View>
```

Comment:
Why did you add this `View` component. It adds additional wrapper which broke some ui.

Reply:
@sayres @malashkevich please check my comment

Reply:
maybe there is should be `Fragment` ?